tests/utils.go

```diff
@@ -2978,6 +2978,142 @@ func newISCSIPVC(name string, size string, accessMode k8sv1.PersistentVolumeAcce
 	}
 }
 
+func CreateNFSTargetPOD() (nfsTargetIP string) {
+	virtClient, err := kubecli.GetKubevirtClient()
+	PanicOnError(err)
+	image := fmt.Sprintf("%s/nfs-server:%s", KubeVirtRepoPrefix, KubeVirtVersionTag)
+	resources := k8sv1.ResourceRequirements{}
+	resources.Limits = make(k8sv1.ResourceList)
+	resources.Limits[k8sv1.ResourceMemory] = resource.MustParse("2048M")
+	hostPathType := k8sv1.HostPathDirectory
+	pod := &k8sv1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "test-nfs-target",
+			Labels: map[string]string{
+				v1.AppLabel: "test-nfs-target",
+			},
+		},
+		Spec: k8sv1.PodSpec{
+			RestartPolicy: k8sv1.RestartPolicyNever,
+			Volumes: []k8sv1.Volume{
+				{
+					Name: "nfsdata",
+					VolumeSource: k8sv1.VolumeSource{
+						HostPath: &k8sv1.HostPathVolumeSource{
+							Path: "/tmp/hostImages/alpine",
```

Comment:
/tmp/hostImages should use os.TempDir()

```
workDir, err = ioutil.TempDir("", tests.TempDirPrefix+"-")
```

Perhaps pass this hostImages directory into the `CreateNFSTargetPOD` function so that test cases can properly clean up after each run?

Reply:
This is a path based on mounts of disk-images-provider, which puts disk images into that dir. So in this specific case I think using TempDir() isn't a good idea.